src/storage/storage.cc

```diff
@@ -337,15 +337,26 @@ Status Storage::Open(DBOpenMode mode) {
   propagate_opts.disable_auto_compactions = config_->rocks_db.disable_auto_compactions;
   SetBlobDB(&propagate_opts);
 
+  rocksdb::ColumnFamilyOptions lite_opts(subkey_opts);
+  if (config_->lite_mode) {
```

Comment:
Seems the logic is write twice? (generate a lite_opts and update all previous opts)

Reply:
Because the lite_opts need to inherit all of subkey_opts properties, so create a `lite_opts` using `lite_opts(subkey_opts)` ?